susePublicCloudInfoClient/lib/susepubliccloudinfoclient/infoserverrequests.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2015 SUSE Linux GmbH.  All rights reserved.
+#
+# This file is part of susePublicCloudInfoClient
+#
+# susePublicCloudInfoClient is free software: you can redistribute it
+# and/or modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation, either version 3 of
+# the License, or (at your option) any later version.
+#
+# susePublicCloudInfoClient is distributed in the hope that it will
+# be useful, but WITHOUT ANY WARRANTY; without even the implied warranty
+# of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with susePublicCloudInfoClient. If not, see
+# <http://www.gnu.org/licenses/>.
+#
+
+import requests
+
+
+def __form_url(
+        framework,
```

Comment:
maybe you could consider to make this a class:

```
class InfoServerRequests:
```

just an idea to have a namespace for this collection